.github/workflows/helm-release.yaml

```diff
@@ -41,8 +41,6 @@ jobs:
 
       - name: Set up Helm
         uses: azure/setup-helm@v3.5
-        with:
-          version: v3.9.2
```

Comment:
this is the only thing im concerned about - should we just default to the latest (stable) helm version here?